domains/iamnotnar.json

```
@@ -0,0 +1,10 @@
+{
+  "owner": {
+    "username": "iamnotnar",
+    "discordId": "1175881305469235321",
+    "discord": "VnVFzNggwt"
+  },
+  "record": {
+    "TXT": "351773beb3717122300722aeb5d1ac"
```

```suggestion
    "TXT": "351773beb3717122300722aeb5d1ac"
```
TXT records alone do nothing. A full and accessible website relating to software development and complying with our [terms of service](https://is-a.dev/terms) is required.